lib/src/editor/ui/editor_tabs.dart

```diff
@@ -91,41 +90,53 @@ class _CoreContentTabsState extends State<CoreContentTabs> {
             ),
           ),
           const TabsVerticalDivider(),
-          StyledTextButton(
-            onPressed: context.read<ZoomProvider>().zoomIn,
-            icon: FeatherIcons.zoomIn,
-          ),
-          const SizedBox(width: 8),
-          StyledTextButton(
-            onPressed: context.read<ZoomProvider>().zoomOut,
-            icon: FeatherIcons.zoomOut,
-          ),
-          const SizedBox(width: 8),
-          StyledTextButton(
-            onPressed: context.read<ZoomProvider>().resetZoom,
-            icon: FeatherIcons.refreshCcw,
-          ),
-          StyledTextButton(
-            onPressed: context.read<GridProvider>().toggleGrid,
-            icon: FeatherIcons.grid,
-          ),
-          const TabsVerticalDivider(),
-          const SizedBox(width: 8),
+          if (context.read<TabProvider>().tab == FlutterBookTab.canvas)
+            _CanvasTabs(),
           StyledTextButton(
             onPressed: context.read<DarkThemeProvider>().toggleDarkTheme,
             icon: FeatherIcons.moon,
           ),
-          const SizedBox(width: 8),
-          StyledTextButton(
-            onPressed: context.read<DevicePreviewProvider>().togglePreview,
-            icon: FeatherIcons.smartphone,
-          ),
         ],
       ),
     );
   }
 }
 
+class _CanvasTabs extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      children: [
+        StyledTextButton(
+          onPressed: context.read<ZoomProvider>().zoomIn,
+          icon: FeatherIcons.zoomIn,
+        ),
+        const SizedBox(width: 8),
```

Comment:
⛏️  - this could be replaced with `_spacer` where 
`const _spacer = SizedBox(width: 8)` for each instance.